update_ip/services/cloudflare.py

```diff
@@ -9,13 +9,16 @@
 class CloudflareService(BaseDNSService):
     name = 'Cloudflare'
 
-    def __init__(self, email, api_key, domain, dns_type, **kwargs):
+    def __init__(self, email, api_key, dns_type, **kwargs):
```

Comment:
There's something I don't understand - how does this module guess which domain you want to update?
Can you only have one domain per api_key?